src/web/app/pages-student/student-home-page/student-home-page.component.scss

```diff
@@ -1,3 +1,9 @@
 .margin-0 {
   margin: 0;
 }
+
+.alignment-buttons {
+  width: 100px;
+  height: 55px;
+  margin-bottom: 5px;
+}
```

Comment:
Was thinking if we could follow the same format as the buttons in the instructor pages:
- Set the styles to apply to a `max-width` of `768px`
- Remove the `height` if not necessary
- Change the name to `.custom-button` or something more descriptive if possible (with corresponding change in the `.html` file)

The above change also allows us to remove the misalignment by adding `margin-right: 0;` for mobile screens